docs/openfaas-cloud/secrets.md

```diff
@@ -0,0 +1,45 @@
+OpenFaaS Cloud **secret** objects lets you store and manage sensitive information securely
+
+
+## Using secrets
+
+Secrets in OpenFaaS Cloud will be mounted as data volumes and made accessible to your function
+
+### Creating secrets
+
+To consume a **secret** in a function:
+
+1.\ Create the secret object `secrets.yml` using literals or from a file
```

Comment:
This is not a secret object. It's a SealedSecret

Reply:
@alexellis isn't it a kubernetes secret object by the time you consume it in your function?